mirdata/download_utils.py

```diff
@@ -57,20 +53,59 @@ def downloader(
     if not os.path.exists(save_dir):
         os.makedirs(save_dir)
 
-    if zip_downloads is not None:
-        for zip_download in zip_downloads:
-            download_zip_file(zip_download, save_dir, force_overwrite, cleanup)
-
-    if tar_downloads is not None:
-        for tar_download in tar_downloads:
-            download_tar_file(tar_download, save_dir, force_overwrite, cleanup)
-
-    if file_downloads is not None:
-        for file_download in file_downloads:
-            download_from_remote(file_download, save_dir, force_overwrite)
-
-    if info_message is not None:
-        print(info_message)
+    if download is not None:
+        if partial_download is not None:
+            if type(partial_download) != list:
+                raise TypeError(
+                    'partial_download should be a list of keys to access the elements of the download dictionary, '
```

Comment:
`'partial_download must be a list which is a subset of {}'.format(download.keys()`